File: Java/string_duplicates.java
//Remove Duplicates
import java.util.Scanner;

public class string_duplicates {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String str = sc.nextLine();

        String result = "";
        for(char ch : str.toCharArray()) {
            if(result.indexOf(ch) == -1) result += ch;
        }
        System.out.println("String after removing duplicates: " + result);
    }
}

File: Java_Practice/CPP/string_duplicates.cpp
//Remove Duplicates
#include <iostream>
#include <string>
using namespace std;

int main() {
    string str;
    cout << "Enter a string: ";
    getline(cin, str);

    string result = "";
    for(char ch : str) {
        if(result.find(ch) == string::npos)
            result += ch;
    }
    cout << "String after removing duplicates: " << result << endl;
    return 0;
}
